contracts/src/v0.8/automation/dev/test/AutomationRegistry2_3.t.sol

```diff
@@ -443,3 +504,133 @@ contract SetConfig is SetUp {
     return bytes32((prefix & prefixMask) | (h & ~prefixMask));
   }
 }
+
+contract NOPsSettlement is SetUp {
+  event NOPsSettledOffchain(address[] payees, uint256[] balances);
```

Comment:
nitpick on name, but maybe "payments" instead of "balances"?

Reply:
done